Apps/Database/src/Assets/docs/CreateDelegationInvite.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Create Delegation Invite</title>
+    </head>
+    <body style="margin: 0">
+        <table
+            style="
+                width: 100%;
+                border-spacing: 0px;
+                margin: 0;
+                color: #707070;
+                font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
+                font-size: 12px;
+            "
+            aria-describedby="Layout Table"
```

Comment:
`aria-describedby` is not appropriate here (it references the id of an element).  However, instead of applying a specific label or description to the table which might be read out, I think this would be more appropriately handled by informing screen readers that the table is used just for presentation and not as a table.

```suggestion
            role="presentation"
```